src/test/groovy/org/prebid/server/functional/model/config/PreferredMediaType.groovy

```diff
@@ -0,0 +1,13 @@
+package org.prebid.server.functional.model.config
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import groovy.transform.ToString
+import org.prebid.server.functional.model.response.auction.MediaType
+
+@ToString(includeNames = true, ignoreNulls = true)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy)
+class PreferredMediaType {
+
+    MediaType generic
```

Comment:
It's per-bidder? Can we express it in a somewhat different way? Or at least make it obvious that it's a bidder. Dumb option would be to add some other bidders here. Other option is to make `AccountAuctionConfig` field for this a map of bidder to preferred media type.